pycomlink/processing/pytorch_utils/run_inference.py

```diff
@@ -0,0 +1,204 @@
+"""
+CNN inference pipeline for CML wet/dry classification.
+
+This module provides functions to run inference on Commercial Microwave Link (CML) 
+data using trained CNN models. The main workflow includes creating sliding windows 
+from time series data, running batch inference, and redistributing predictions back 
+to the original data structure.
+
+Main Functions:
+    - cnn_wd(): High-level interface for wet/dry classification
+    - run_inference(): Core inference pipeline 
+    - rolling_window(): Creates sliding windows from time series
+    - redistribute_results(): Maps predictions back to original grid
```

Comment:
can you adjust the module level docstring to the actual content? I think this should be "back to original coordinates" and the run_id option (below) is not supported